# Disables compiling the syn123 library because we don't use this library and it fails to compile for Emscripten.

--- a/ports/cmake/src/CMakeLists.txt
+++ b/ports/cmake/src/CMakeLists.txt
@@ -370,7 +370,7 @@ add_subdirectory("libmpg123")
 if(BUILD_LIBOUT123)
     add_subdirectory("libout123")
 endif()
-add_subdirectory("libsyn123")
+
 
 if(UNIX AND NOT PORTABLE_API)
     option(BUILD_PROGRAMS "Build programs (mpg123 executable and others)" ON)
